go.mod

```diff
@@ -36,4 +36,5 @@ require (
 	gopkg.in/mgo.v2 v2.0.0-20190816093944-a6b53ec6cb22
```

Comment:
Added client-go and API structures from kubernetes.  Many of the structs we have are partial structures for kubernetes APIs, by using the official kube structures we get more for less code.